### DISCUSSION TITLE
Windows Package Manager v0.4.11391 Preview

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#975] [Releases]
This release includes improvements for the correlation between manifests in the community repository and entries in Add / Remove Programs. We've also added a hash for the manifest itself to ensure the chain of trust with the PreIndexed package (default source). Several features have moved from experimental to released including "list", "upgrade", "uninstall", and "export".

# New Features
#959 Ensure chain of trust with packaged index
#118 Repository REST API
#120 Upgrade an installed App
#220 Export/Import apps list
#121 Uninstall an Installed App
#119 List installed Apps
#151 Support Delivery Optimization

# Bug Fixes
#723 commands not showing with white cmd background
#955 Name and Publisher Correlation in CompositeSource
#966 Fix a few polish Items
#973 Fix for a bug, DO timeout to 60 seconds, minor version bump

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v-0.4.11391-preview'>Windows Package Manager v0.4.11391 Preview</a>.</em>

**Comment by psychelzh** [comment]
Seems that upgrade does not work very well. It always tries to upgrade some of my applications: e.g., Kindle for PC, CPU-Z, VS Code, etc.

**Reply by denelon** [reply]
We are seeing discrepancies between the version reported in the manifest and the version reported in Add / Remove Programs. #976 was the first report for Visual Studio Code. A few other packages have been reported there as well. Feel free to add any other packages impacted as we work towards a solution.
